boot/freeldr/freeldr/ntldr/wlmemory.c

```diff
@@ -238,6 +239,42 @@ WinLdrSetupMemoryLayout(IN OUT PLOADER_PARAMETER_BLOCK LoaderBlock)
         return FALSE;
     }
 
+    // Always map the ACPI table if present, otherwise Windows will crash.
+    if (Extension->AcpiTable)
+    {
+        PDESCRIPTION_HEADER AcpiTable = Extension->AcpiTable;
+        PRSDT_DESCRIPTOR Rsdt = Extension->AcpiTable;
+        PXSDT_DESCRIPTOR Xsdt = Extension->AcpiTable;
```

Comment:
It might be possible to unionize these:
```suggestion
        union
        {
            PDESCRIPTION_HEADER Header;
            PRSDT_DESCRIPTOR Rsdt;
            PXSDT_DESCRIPTOR Xsdt;
        } Table = Extension->AcpiTable;
```
with the replacements:
`AcpiTable->...`  ---> `Table.Header->...`
and `Xsdt->` ---> `Table.Xsdt->` (and similar for Rsdt).